### PR TITLE
Release 2 2 0

### PR DESCRIPTION
**Pull request by tunniclm**
* Rework service.version variable extraction (#216)
* Readme for Alert Notification (#215)
* Improve integration tests for new services (#211)
* Add initial health endpoint (#208)
* Increase mocha default timeout for test stability (#212)
* Fix docker using root user on Linux (#185)
* Improve support for debugging with docker (#210)
* Fix minor issues in watson tests (#209)
* Add Alert notification service (#206)
* Added Tests folder with a basic test example to common files (#202)
* Add Watson conversation service (#205)

### CODE REVIEW

**Review by tfrank64 — APPROVED** [review]

**Review by rolivieri — APPROVED** [review]

File: refresh/templates/extensions/WatsonConversationExtension.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Foundation
+import ConversationV1
+import CloudFoundryConfig
+
+extension Conversation {
+
+   public convenience init(service: WatsonConversationService) {
+
+        let version = "<%- service.version %>"
```
Comment (rob-deans) [inline refresh/templates/extensions/WatsonConversationExtension.swift]: There was a bug with this line not getting run through a template - did it get fixed? I think we need to fix this before we can release @jkingoliver

File: refresh/templates/common/README.scaffold.md [hunk 2]
```diff
@@ -166,6 +169,18 @@ Boilerplate code for creating a client object for the App ID API is included ins
...
+This application uses the [Watson Swift SDK package](https://github.com/watson-developer-cloud/swift-sdk), which allows Kitura applications to build Watson-powered applications, specifically in this case the IBM Watson Conversation service.
+
+With the IBM Watson Conversation service you can create cognitive agents--virtual agents that combine machine learning, natural language understanding, and integrated dialog scripting tools to build outstanding projects, such as a chat room with an integrated Watson chat bot.
+
+Boilerplate code for creating a client object for the Watson Conversation API is included inside `Sources/Application/Application.swift` as an `internal` variable available for use anywhere in the `Application` module.
+
+The connection details for this client are loaded by the [configuration](#configuration) code and are passed to the Watson Conversation client in the boilerplate code.
+
+More information about the Watson Conversation can be found in the [README](https://github.com/watson-developer-cloud/swift-sdk#conversation)
+<%   } -%>
```
Comment (rob-deans) [inline refresh/templates/common/README.scaffold.md]: Do we want to add a README snippet for Alert Notification service in the README scaffold before we release? @jkingoliver

File: refresh/templates/extensions/WatsonConversationExtension.swift (hunk 1 above)
Comment (tunniclm) [reply refresh/templates/extensions/WatsonConversationExtension.swift]: This issue, I would guess: https://github.com/IBM-Swift/generator-swiftserver/issues/207

File: refresh/templates/common/README.scaffold.md (hunk 2 above)
Comment (jkingoliver) [reply refresh/templates/common/README.scaffold.md]: I'll add a README today

**Comment by jkingoliver** [comment]
Do we want to wait for Push SDK generator to release?

**Comment by tunniclm** [comment]
@jkingoliver only if it won't delay the release for more than a day. We can always make another release.

File: refresh/templates/extensions/WatsonConversationExtension.swift (hunk 1 above)
Comment (jkingoliver) [reply refresh/templates/extensions/WatsonConversationExtension.swift]: #207 is complete

**Comment by tunniclm** [comment]
Rebased with latest on `develop`

**Review by jkingoliver — APPROVED** [review]

**Pull request merged**